src/components/Lobby/index.js

```diff
@@ -189,13 +216,20 @@ export default function Lobby() {
           </RoomsContainer>
           <Chats>
             <ChatsHead>Chats</ChatsHead>
-            <ChatList>
-              <ChatContainer>{displayName}: message</ChatContainer>
-              <ChatContainer>{displayName}: message</ChatContainer>
+            <ChatList ref={chatListRef}>
+              {ReceivedMessages.map(({ user, chat }, index) => (
+                <ChatContainer key={user + chat + index}>
```

Comment:
```suggestion
                <Chat key={user + chat + index}>
```

`<ChatContainer>`와 `<Chats>`의` styled-components` 명이 바뀌어야 할 것 같습니다